platform/android/os_android.cpp

```diff
@@ -406,6 +409,35 @@ void OS_Android::process_touch(int p_what, int p_pointer, const Vector<TouchPos>
 
 			ERR_FAIL_COND(touch.size() != p_points.size());
 
+			if (touch.size() == 1) {
+				Point2 d = (p_points[0].pos - touch.write[0].pos);
+				if (fabs(d.x) > PAN_GESTURE_MIN_DELTA || abs(d.y) > PAN_GESTURE_MIN_DELTA) {
```

Comment:
Should `abs(d.y)` be `fabs(d.y)`?

Reply:
haaaaaa, shame on my family, damn you're good